db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/WindowResultSetNode.java

```diff
@@ -779,12 +781,12 @@ private void addWindowFunctionColumns() throws StandardException {
                         LOG.debug("addWindowFunctionColumns() operand does not have an RC: " + exp.getColumnName() +
                                       ". Creating one for it in " + wdn.toString());
                     }
+                    exp.accept(columnPuller);
 
                     tmpRC = (ResultColumn) getNodeFactory().getNode(C_NodeTypes.RESULT_COLUMN,
-                                                                           "##" + windowFunctionNode.getAggregateName() +
-                                                                               "_Input_" + exp.getColumnName(),
-                                                                           exp,
-                                                                           getContextManager());
+                             "##" + windowFunctionNode.getAggregateName() + "_Input_" +
```

Comment:
Can you move `"##" + windowFunctionNode.getAggregateName() + "_Input_" +...` to a separate local value, say `name` or `id`?